integrations/tensorflow/iree_tf_compiler/iree-import-tflite-main.cpp

```diff
@@ -37,6 +44,15 @@ int main(int argc, char **argv) {
                                              cl::value_desc("filename"),
                                              cl::init("-"));
 
+  // The output format flag is the master control for what we do with the
+  // in-memory compiled form.
+  llvm::cl::opt<OutputFormat> outputFormat(
+      "output-format", llvm::cl::desc("Format of imported output"),
+      llvm::cl::values(clEnumValN(OutputFormat::mlir_bytecode, "mlir-bytecode",
+                                  "MLIR Bytecode (default)"),
+                       clEnumValN(OutputFormat::mlir_ir, "mlir-ir", "MLIR IR")),
+      llvm::cl::init(OutputFormat::mlir_bytecode));
```

Comment:
For tools which can output multiple formats, sometimes the output file name can also be used to infer the output format. We could have `-o *.mlir` imply `--output-format=mlir-ir` and `-o *.mlirbc` imply `--output-format=mlir-bytecode`, with all other output names (including stdout) using the default value for this flag.

That would have helped some of our build system scripts that invoke these tools with file names: https://github.com/iree-org/iree/blob/97820614d2b6565d0f5a8b2fecd48fc947add518/build_tools/cmake/iree_benchmark_suite.cmake#L38-L47

Now we should go through and change those `OUTPUT_MLIR_FILE` variables to `.mlirbc` instead of `.mlir`